Alignment/OfflineValidation/plugins/PrimaryVertexValidation.cc

```diff
@@ -1173,9 +1173,9 @@ bool PrimaryVertexValidation::isHit2D(const TrackingRecHit& hit, const PVValHelp
         else if (dynamic_cast<const ProjectedSiStripRecHit2D*>(&hit))
           return false;  // crazy hit...
         else {
-          edm::LogError("UnkownType") << "@SUB=PrimaryVertexValidation::isHit2D"
-                                      << "Tracker hit not in pixel and neither SiStripRecHit2D nor "
-                                      << "SiStripMatchedRecHit2D nor ProjectedSiStripRecHit2D.";
+          edm::LogError("UnknownType") << "@SUB=AlignmentTrackSelector::isHit2D"
```

Comment:
Why has this been modified from `PrimaryVertexValidation::isHit2D`? At https://github.com/cms-sw/cmssw/pull/32432/files#diff-2b4e53d29d0035ce6ff0c0f3ca63fcc8c860fcd91a70bd68bdf4e1fd491031cbR2574